includes/models/model.llms.lesson.php

```diff
@@ -720,190 +719,216 @@ public function has_content() {
 		}
 	}
 
+
+
+
+
 	/**
 	 * Get next lesson ID
 	 *
 	 * @since 1.0.0
 	 * @since 3.24.0
 	 * @since 4.4.0 Improve query so that unpublished siblings do not break expected results.
-	 * @since [version] Use a numeric comparison for the next position meta query.
+	 * @since [version] Use a numeric comparison for the previous position meta query.
+	 * @since [version] Refactor to use helper method `get_sibling()`.
 	 *
-	 * @return false|int ID of the next lesson, if any, `false` otherwise.
+	 * @return false|int WP_Post ID of the next lesson or `false` if one doesn't exist.
 	 */
 	public function get_next_lesson() {
 
-		// phpcs:disable WordPress.DB.SlowDBQuery.slow_db_query_meta_key
-		// phpcs:disable WordPress.DB.SlowDBQuery.slow_db_query_meta_query
-		// phpcs:disable WordPress.WP.PostsPerPage.posts_per_page_posts_per_page
+		return $this->get_sibling( 'next' );
+
+	}
+
+	/**
+	 * Get previous lesson ID
+	 *
+	 * @since 1.0.0
+	 * @since 3.24.0 Unknown.
+	 * @since 3.24.0 Unknown.
+	 * @since 4.4.0 Improve query so that unpublished siblings do not break expected results.
+	 *              Use strict comparisions where needed.
+	 *              Make sure to always return `false` if no previous lesson is found.
+	 * @since [version] Use a numeric comparison for the previous position meta query.
```

Comment:
```suggestion
	 * @since 4.4.2 Use a numeric comparison for the previous position meta query.
```

This PR needs to be rebased to the current trunk, so I don't even know whether I should bother you with this kind of change request.